src/ecosystem/connectivity.jl

```diff
@@ -130,9 +130,8 @@ function site_connectivity(
         extracted_conn[extracted_conn .< conn_cutoff] .= 0.0
     end
 
-    conn = NamedDimsArray(
-        extracted_conn; Source=loc_ids, Sink=loc_ids
-    )
+    conn = YAXArray((Dim{:Source}(loc_ids), Dim{:Sink}(loc_ids)), extracted_conn)
```

Comment:
Maybe standardize? Don't really mind if you leave this as is.

```suggestion
    conn = DataCube(extracted_conn; Source=loc_ids, Sink=loc_ids)
```